lib/rom/repository/command_proxy.rb

```diff
@@ -11,11 +11,10 @@ class CommandProxy
 
       def initialize(command)
         @command = command
-        @root = Dry::Core::Inflector.singularize(command.name.relation).to_sym
       end
 
       def call(input)
-        command.call(root => input)
+        command.call(command.name.relation => input)
       end
```

Comment:
It is probably not that difficult to tweak the code in core so that we can ask graph builder to build the command where the first one is marked as a root without a tuple path. Basically [here](https://github.com/rom-rb/rom/blob/master/lib/rom/commands/graph/class_interface.rb#L44) we ALWAYS append a key to the path (and path is [empty by default](https://github.com/rom-rb/rom/blob/master/lib/rom/commands/graph/class_interface.rb#L21)) that's how we end up with evaluator that always expects a tuple path. So we'd have to change two things:

1) make it possible to do something like `ROM::Commands::Graph.build(registry, graph_opts, false` which we'd call right [here](https://github.com/rom-rb/rom-repository/blob/master/lib/rom/repository/command_compiler.rb#L46)
2) tweak these [two lines](https://github.com/rom-rb/rom/blob/master/lib/rom/commands/graph/class_interface.rb#L44-L45) so that if `path` is `false` then we use an input evaluator that will simply return the input (literally `-> input { input }`)